be/src/service/doris_main.cpp

```diff
@@ -74,6 +79,199 @@ static void thrift_output(const char* x) {
 
 } // namespace doris
 
+// These code is referenced from clickhouse
```

Comment:
Can these code be moved to a separated file. like `util/instruction_utils.hpp`?

Reply:
1. These code is not utils, they will be only used in the main function and could not in other functions. There maybe some signal catching problem if running in other functions I have not test it.
2. These code should run at the beginning of the program so I add __attribute__(priority = 101) to checker. If other functions running before these code meet SIMD code and could not find the SSE instruction the program may crash we could not catch the signal.

Reply:
Ok I see.
I've checked and it works.